src/OmniSharp.Roslyn.CSharp/project.json

```diff
@@ -6,9 +6,9 @@
   "dependencies": {
     "OmniSharp.Abstractions": "1.0.0",
     "OmniSharp.Roslyn": "1.0.0",
-    "Microsoft.CodeAnalysis.CSharp": "1.3.0",
-    "Microsoft.CodeAnalysis.CSharp.Features": "1.3.0",
-    "Microsoft.CodeAnalysis.CSharp.Workspaces": "1.3.0"
+    "Microsoft.CodeAnalysis.CSharp": "2.0.0-beta3",
+    "Microsoft.CodeAnalysis.CSharp.Features": "2.0.0-beta3",
```

Comment:
I'm assuming the internal class used for find metadata hasn't changed, but do you know if it has?